pkg/ipmi/operation.go

```diff
@@ -66,6 +66,10 @@ var (
 		Function: NetworkFunctionAppRsp,
 		Command:  0x38,
 	}
+	OperationSetSessionPrivilegeLevelReq = Operation{
```

Comment:
Don't forget to add another operation for the response, and add that to the `operationLayerTypes` map. I can't remember why I thought it was necessary to specify the response layer type in two places - I'll have to look into that.